airflow/providers/databricks/hooks/databricks.py

```diff
@@ -352,3 +355,50 @@ def get_repo_by_path(self, path: str) -> Optional[str]:
             return str(result['object_id'])
 
         return None
+
+    def import_notebook(self, dbfs_path: str, raw_code: str, language: str, overwrite: bool = True):
+        """
+        Import a local notebook from Airflow into Databricks FS. Notebooks saved to /Shared/airflow dbfs
```

Comment:
it's incorrect sentence - notebooks aren't stored on DBFS, they are stored in the workspace. To work with DBFS there is an another API

Reply:
@pohek321 do you still work on this? I have notebook operators in the TODO list